modules/gapi/include/opencv2/gapi/gmetaarg.hpp

```diff
@@ -61,6 +61,16 @@ namespace detail
 
 } // namespace detail
 
+// Forward declaration for gmat
```

Comment:
not a gmat but a `Mat` and `UMat` . And IMHO this comment is excessive and I would remove it.